web/src/modules/app/containers/Switcher.js

```diff
@@ -49,7 +51,7 @@ type PropsType = {|
   sprungbrettJobsPayload: Payload<Array<SprungbrettOfferPage>>,
   wohnenOffersPayload: Payload<Array<WohnenOfferModel>>,
   disclaimerPayload: Payload<PageModel>,
-  languages: ?Array<LanguageModel>,
+  languagesPayload: Payload<Array<LanguageModel>>,
```

Comment:
Why did you change this? Why is Payload now necessary or why wasn't if before?

Reply:
Good question. In fact, it is not really necessary now as it is only checked on the data anyway. However, I though we need to check for errors/loading of the languages as well, which was not the case. Left it like this for consistency though.